tests/scene/test_code_edit.h

```diff
@@ -905,8 +905,7 @@ TEST_CASE("[SceneTree][CodeEdit] delimiters") {
 			CHECK(code_edit->get_string_delimiters().size() == 2);
 
 			/* Set should override existing, and test multiline */
-			TypedArray<String> delimiters;
-			delimiters.push_back("^^ ^^");
+			TypedArray<String> delimiters(Array{ "^^ ^^" });
```

Comment:
Are we able to use the initializer list for these variables too?

Reply:
I see the discussion above, but I'm not sure if that was before you added the initializer_list constructor and I've seen other instances of TypedArray using the initializer

Reply:
I didn't find more of them.